### PR TITLE
Improvements/global

### PR DESCRIPTION
**Pull request by djerfy**
## Description

Global improvements:
* Fix syntax (i18n, space)
* Add INVALID_AGG_SIGNATURE error partial (without details for the moment)
* Reduce images size (Partners page)

## Test(s)

From localhost only

And through the environments (browser):

- [x] Desktop
- [ ] Tablet
- [ ] Mobile

## Screenshot(s)

No

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/partners/partners.component.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h5 class="mb-0">
                       aria-expanded="false" aria-hidden="false">
                       <br/>
                       <p>
-                        <a href="https://spacescan.io" target="_blank"><b>Spacescan.io</b></a> <span i18n>is new generation Chia blockchain 
+                        <a href="https://spacescan.io" target="_blank"><b>Spacescan.io</b></a><span i18n> is new generation Chia blockchain
```
Comment (william-gr) [inline src/app/partners/partners.component.html:48]: wouldnt the version without space be better?

Comment (djerfy) [reply src/app/partners/partners.component.html:48]: I don't know, else I don't have space between Spacescan link and the description:

image

Comment (william-gr) [reply src/app/partners/partners.component.html:48]: you can use `&nbsp;`

Comment (djerfy) [reply src/app/partners/partners.component.html]: Thx, and update done (it's better). Also margin right has been added.

**Review by william-gr — APPROVED** [review]
Thanks!

**Pull request merged**